app/lib/constants.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
     PERIOD_END: 'PeriodEnd',
     SUPPLIER: 'Supplier',
   },
-  OUTPUT_DIR: 'output',
+  OUTPUT_DIR: 'html/json',
```

Comment:
Will only contain one file format once issue https://github.com/nhsuk/pomi-data-etl/issues/20 is addressed.